packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -44,6 +47,24 @@ function genConfig(compatVersion: number): CompatConfig[] {
         }];
     }
 
+    if (typeof compatVersion === "string") {
```

Comment:
This test works great for defaults, but I wonder how it would work when using fluid__test__compatVersion envirnment variable (see lines ~163), where all args (I'd guess) would be strings. Should we test if it's non-positive integer in string form, and if so - use the default path? Or maybe this is already done in some way? Not sure how we were getting integers here if fluid__test__compatVersion was used. Can you please check?

Reply:
This is a condition to see if the compatVersion is a LTS version, right. It might better to see if compatVersion is in the LTSVersions array?